src/main/java/uk/gov/hmcts/sptribs/ciccase/model/State.java

```diff
@@ -98,6 +98,13 @@ public enum State {
     )
     ConsentOrder("ConsentOrder"),
 
+    @CCD(
+        label = "Death of Appellant",
+        hint = "### ${cicCaseFullName}\nCase number: ${hyphenatedCaseRef}",
+        access = {DefaultStateAccessExcludingCAA.class}
+    )
+    DeathOfAppellant("DeathOfAppellant"),
```

Comment:
I think only the case close reason has been changed. The state should still be CaseClosed.

Reply:
It is the design approach the old team used for the reason. The state will not be used